documentation/asciidoc/topics/proc_configuring_autoscaling.adoc

```diff
@@ -0,0 +1,30 @@
+[id='configuring_autoscale-{context}']
+= Configuring Automatic Scaling
+When you create clusters with {cacheservice} nodes, {ispn_operator} can
+automatically scale up or down based on memory usage.
+
+{ispn_operator} monitors memory usage for each node in the {brandname} cluster.
+If it detects that the {cacheservice} cluster requires additional capacity,
+{ispn_operator} creates new nodes rather than evicting entries. Likewise, if it detects that memory usage is below a certain threshold, {ispn_operator} shuts down nodes.
+
+.Procedure
+
+. Add the `spec.autoscale` resource to your **Infinispan** CR to enable automatic scaling.
+. Configure memory usage thresholds and number of replicas to control when {ispn_operator} automatically scales nodes up or down.
+. Apply the changes.
+
+----
+spec:
+  ...
+  autoscale: <1>
+    maxMemUsagePercent: 70 <2>
+    maxReplicas: 5 <3>
+    minMemUsagePercent: 30 <4>
+    minReplicas: 2 <5>
+----
+
+<1> Enables automatic scaling.
+<2> Configures the maximum threshold, as a percentage, for memory usage on each node. When {ispn_operator} detects that any node in the cluster reaches the threshold, it creates a new node if possible. If {ispn_operator} cannot create a new node then it performs eviction when memory usage reaches 100 percent.
+<3> Defines the maximum number of replicas for each cache entry across the cluster.
+<4> Configures the minimum threshold, as a percentage, for memory usage across the cluster. When {ispn_operator} detects that memory usage falls below the minimum, it shuts down nodes.
+<5> Defines the minimum number of replicas for each cache entry across the cluster.
```

Comment:
@rigazilla are there any defaults for autoscale? can you just add "autoscale" to the CR without specifying thresholds?

I'd also like to expand on the max and min replica fields here. it isn't quite clear to me how these affect autoscale.

Reply:
`maxReplicas` and `minReplicas` define the max/min number of pods the autoscaler is allowed to start. They aren't related to the owners/replicationFactor which is a different thing.